production/helm/loki/values.yaml

```diff
@@ -1643,6 +1643,8 @@ backend:
   extraEnv: []
   # -- Environment variables from secrets or configmaps to add to the backend pods
   extraEnvFrom: []
+  # -- Lifecycle for the backend container
+  lifecycle: {}
```

Comment:
do we need to add a `lifecycle` property for the `read` pod as well?

Reply:
We want to add sleep to ensure that the iptables rules are updated during the termination process so that requests are not lost.

Reply:
the question was about the `values.yaml`, as you only seemed to be adding it to the `backend`, and not `read`, but it looks as if it's already an option for `read`